SpatialGDK/Source/SpatialGDK/Private/Schema/UnrealObjectRef.cpp

```diff
@@ -40,6 +40,14 @@ UObject* FUnrealObjectRef::ToObjectPtr(const FUnrealObjectRef& ObjectRef, USpati
 			UObject* Value = PackageMap->GetObjectFromNetGUID(NetGUID, true);
 			if (Value == nullptr)
 			{
+				// Check if the object we are looking for is in a package being loaded.
+				if (PackageMap->IsGUIDPending(NetGUID))
+				{
+					PackageMap->PendingReferences.Add(NetGUID);
+					bOutUnresolved = true;
```

Comment:
It seems like this parameter is always set to true when the return is nullptr. So I'm not really sure what the point of it is. Might be easier just to remove it.

Reply:
Not if the NetGUID is valid, unresolved is left to false. I think it means "we were not able to resolve a stably named asset, no chance we'll get it in the future"

Reply:
Well, this is an awkward pattern that is going on throughout the serialization code. The boolean is a reference shared by all fields of a property. It starts false, and anyone can set it to true while digging into structs, or iterating over an array. If I set it to false, I can stomp on some other field that set it to true.

Reply:
I see, coooolio.